test/functionalTest/cases/3112_ngsiv1_autocast/update_attributes_with_autocast.test

```diff
@@ -0,0 +1,328 @@
+Date: REGEX(.*)
+
+{
+    "B": {
+        "metadata": {},
+        "type": "Boolean",
+        "value": "truex"
+    },
+    "D": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "20160112T15:25:00.00Z"
+    },
+    "I": {
+        "metadata": {},
+        "type": "ISO8601",
+        "value": "year-02-24T10:00:00.00Z"
+    },
+    "N": {
+        "metadata": {},
+        "type": "Number",
+        "value": "-2x3"
+    },
+    "Q": {
+        "metadata": {},
+        "type": "Quantity",
+        "value": "11,,31"
+    },
+    "id": "E",
+    "type": "T"
+}
+
+
+03. Update attributes with 'right' Quantity, Number, Boolean, DateTime and ISO8601 values
+=========================================================================================
+HTTP/1.1 200 OK
+Content-Length: 352
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "N",
+                        "type": "Number",
+                        "value": ""
+                    },
+                    {
+                        "name": "Q",
+                        "type": "Quantity",
+                        "value": ""
+                    },
+                    {
+                        "name": "B",
+                        "type": "Boolean",
+                        "value": ""
+                    },
+                    {
+                        "name": "D",
+                        "type": "DateTime",
+                        "value": ""
+                    },
+                    {
+                        "name": "I",
+                        "type": "ISO8601",
+                        "value": ""
+                    }
+                ],
+                "id": "E",
+                "isPattern": "false",
+                "type": "T"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+04. Get entity using NGSIv2 and check values (right types)
+==========================================================
+HTTP/1.1 200 OK
+Content-Length: 315
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "B": {
+        "metadata": {},
+        "type": "Boolean",
+        "value": false
+    },
+    "D": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2018-01-12T15:25:00.00Z"
+    },
+    "I": {
+        "metadata": {},
+        "type": "ISO8601",
+        "value": "2019-02-24T10:00:00.00Z"
+    },
+    "N": {
+        "metadata": {},
+        "type": "Number",
+        "value": 11
+    },
+    "Q": {
+        "metadata": {},
+        "type": "Quantity",
+        "value": -41.31
+    },
+    "id": "E",
+    "type": "T"
+}
+
+
+--TEARDOWN--
+brokerStop CB
+#dbDrop CB
```

Comment:
Look, just like I use to leave the functests :-D

Reply:
Fixed in 4cc8459